lib/ts/BufferWriterForward.h

```diff
@@ -94,7 +94,7 @@ BWFSpec::is_sign(char c)
 inline bool
 BWFSpec::is_type(char c)
 {
-  return 'x' == c || 'X' == c || 'o' == c || 'b' == c || 'B' == c || 'd' == c;
+  return 'x' == c || 'X' == c || 'o' == c || 'b' == c || 'B' == c || 'd' == c || 's' == c || 'S' == c;
```

Comment:
`s` for `string`. It seemed the best overall choice. I didn't like `b` because it is numeric and might quite reasonably be viewed as requiring a binary output, not a textual one.

Reply:
l for logical, f for flag.